src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1312,7 +1312,7 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
         }
 
         $container->setParameter('validator.auto_mapping', $config['auto_mapping']);
-        if (!$propertyInfoEnabled || !$config['auto_mapping'] || !class_exists(PropertyInfoLoader::class)) {
+        if (!$propertyInfoEnabled || !class_exists(PropertyInfoLoader::class)) {
```

Comment:
Could we clarify in `Configuration` (maybe with an `->info()`) the valid values for `auto_mapping` and what they mean?

* Pass an array to enable auto-mapping & activate it automatically for matching namespaces
* Pass null to turn the auto-mapping feature on only and activate with config

The wording/explanation might be more clear if we (in this PR) add a proper enable/disable flag to this config. And maybe instead of `auto_mapping: ~` we make it be `auto_mapping: true`.